tests/test_mathematical.py

```diff
@@ -965,9 +965,9 @@ def test_invalid_out(self, out):
         assert_raises(TypeError, numpy.add, a.asnumpy(), 2, out)
 
 
-class TestHypot:
-    @pytest.mark.parametrize("dtype", get_float_dtypes())
-    def test_hypot(self, dtype):
+class TestDivide:
```

Comment:
We already have an in-place test for `floor_divide`.
https://github.com/IntelPython/dpnp/blob/master/tests/test_mathematical.py#L1748

Reply:
I wonder how it works before the PR in dpctl then and if it fully works why we need to change something.

Reply:
As I understand it, we have an in-place operators for all of the functions which include an intermediate steps of allocating an additional buffer for the output and then copying it back to the first operand.. A few functions have dedicated kernels for in-place operator (`add`, `multiply`, `subtract`, `divide`, and `floor_divide`) which eliminate this intermediate step. Am I right @ndgrigorian?

Reply:
That's correct. Any function which Python implements an in-place operator for (so ``+=``, ``-=``, etc.) are being given dedicated kernels to avoid allocating a buffer and copying back. It also reduces the number of stride calculations.

So far, ``add``, ``multiply``, ``subtract``, ``divide``, and ``floor_divide`` are done. ``pow``, ``remainder``, ``bitwise_left_shift``, ``bitwise_right_shift``, ``bitwise_or``, ``bitwise_and``, `and `bitwise_xor`` are currently in the pipeline.